apps/trivium/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(clippy::module_inception)]
+#![allow(clippy::should_implement_trait)]
```

Comment:
why is this needed ? 🤔

Reply:
which next function is impacted ?

Reply:
apps/trivium/src/kreyvium/kreyvium_shortint.rs, line 78
apps/trivium/src/kreyvium/kreyvium.rs, line 152
apps/trivium/src/trivium/trivium_shortint.rs, line 66
apps/trivium/src/trivium/trivium.rs, line 124

I'll put the lint on each function

Reply:
otherwise rename the function to "next_byte" or whatever name fits the returned value ?

or implement iterator for those ? I'm really not sure as I don't really know the trivium/kreyvium apps

Reply:
Renamed `next_ct` and `next_bool`